military_time/military_time.rb

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+print 'Enter time (HHMM):'
+military_time = gets.chomp
+begin
+  military_time = Integer military_time
+  if (military_time > 2400) || (military_time % 100 > 59)
+    puts 'Invalid time'
+  elsif military_time < 1259
+    puts "\n non-Military Time:" + military_time.to_s + 'AM'
+  else
+    print "\n non-Military Time:" + (military_time - 1200).to_s + 'PM'
+  end
+rescue
```

Comment:
_Never_ `rescue` without specifying what exception(s) you're catching.
Also, avoid rescuing just `StandardError`. That will catch almost all exceptions, which is not desirable behavior.

Reply:
See https://andycroll.com/ruby/rescue-specific-errors-avoid-standarderror-do-not-rescue-exception/